Dockerfile

```diff
@@ -1,11 +1,10 @@
-FROM centos:7
+FROM registry.access.redhat.com/ubi8/ubi-minimal
 
 LABEL maintainer="ksimon@redhat.com"
 
 ENV container docker
 
-RUN yum -y update
-RUN yum clean all
+RUN microdnf -y update && microdnf clean all
```

Comment:
not sure this is needed, I'm pruning this step from my images for the time being. I think you can keep it if you wish.